app/src/main/java/app/revanced/manager/util/PM.kt

```diff
@@ -80,7 +80,9 @@ class PM(
             (compatibleApps.await() + installedApps.await())
                 .distinctBy { it.packageName }
                 .sortedWith(
-                    compareByDescending<AppInfo> {
+                    compareBy<AppInfo>{
+                        it.packageInfo == null || it.patches == 0
```

Comment:
It might be more readable to use `compareByDescending` instead
```suggestion
                    compareByDescending<AppInfo>{
                        it.packageInfo != null && it.patches > 0
```